src/styles/base/_base.scss

```diff
@@ -0,0 +1,27 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+
+  @extend %font-roboto;
+}
+
+.button {
+  display: block;
+  width: 100%;
+  border: none;
+  cursor: pointer;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+}
+
+a {
+  transition: all 0.3s;
```

Comment:
It's not recommended to use `all` in the `transition` property. This can cause unintended transitions on properties you may not want to transition. It's better to specify which properties you want to transition.